symengine/basic.h

```diff
@@ -227,6 +227,43 @@ inline bool is_true(tribool x)
     return x == tribool::tritrue;
 };
 
+inline bool is_false(tribool x)
```

Comment:
Can you keep the declaration here and move the implementation to `basic-inl.h`? Sorry, I forgot to mention this in `is_true`.

Reply:
Ok!